Gemfile.lock

```diff
@@ -107,7 +107,7 @@ GEM
     faraday_middleware (1.0.0)
       faraday (~> 1.0)
     fastimage (2.2.0)
-    fastlane (2.158.0)
+    fastlane (2.162.0)
```

Comment:
Had to update Fastlane because the `deliver` command didn't work with version 2.158.0.